docs/builds.mdx

```diff
@@ -6,13 +6,16 @@ icon: "hammer"
 
 Shuttle builds run on AWS CodeBuild, where we compile your Rust app and place it in a Shuttle `runtime` Debian-based Docker image.
 
+## Build architecture
+Unless you are on the Growth tier, your projects will primarily be built and run on an Arm64 (aarch64) architecture.
```

Comment:
Perhaps include a link to the pricing page? I also think "primarily" is a little bit ambiguous.
```suggestion
Unless you are on the [Growth tier](https://www.shuttle.dev/pricing), your projects will be built and run on an Arm64 (aarch64) architecture.
```